### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by surya-nemiwal**
added wishlist page that can be accessible from the home page navigation bar.

### CODE REVIEW

File: HomePage/index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>X Mart | Home</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (haarsh24) [inline HomePage/index.html:7]: Suggestion: add meta tags for author and description for better SEO

File: HomePage/index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                  style="color: rgb(226, 17, 17)"
+                  >favorite</span
+                >
+              </i></a
+            >
+            <div class="badge badge-icon">3</div>
+          </div>
+          <a href="../index.html"
+            ><img class="avatar-smallest" src="https://picsum.photos/100"
+          /></a>
```
Comment (haarsh24) [inline HomePage/index.html:53]: Add alt tags in img

File: HomePage/index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            >
+
+            <div class="badge badge-icon">3</div>
+          </div>
+          <div class="icon-with-badge">
+            <a href="../wishlist/wishlist.html"
+              ><i
+                ><span
+                  class="material-icons-outlined"
+                  style="color: rgb(226, 17, 17)"
```
Comment (haarsh24) [inline HomePage/index.html:44]: Don't add inline style

File: HomePage/index.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        <div class="hero-cta" onclick="handleCategoryClick()">Shop Now</div>
+      </div>
+    </section>
+    <h2 class="feature-category-title">Categories To Bag</h2>
+    <section class="feature-category-list">
+      <div
+        class="feature-category-item"
+        onclick="handleCategoryClick('shirts')"
+      >
+        <div class="img-div img-circle" style="height: auto; width: 80%">
```
Comment (haarsh24) [inline HomePage/index.html:72]: Don't add inline styles.

File: HomePage/style.css [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+.feature-category-list {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.feature-category-item {
+  width: 22%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (haarsh24) [inline HomePage/style.css:80]: Add these styles in util , so you can reuse in multiple areas.

File: HomePage/style.css [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+  background-color: rgb(224, 206, 206);
```
Comment (haarsh24) [inline HomePage/style.css:81]: Add colors in root and then use it here.

File: HomePage/style.css [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+  margin-top: 1.5rem;
+}
+
+.category-img {
+  height: 50px;
+}
+
+.hero-img-container {
+  margin-top: 5rem;
+  color: white;
```
Comment (haarsh24) [inline HomePage/style.css:104]: Same here

File: HomePage/style.css [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+  flex-wrap: wrap;
+}
+
+.card{
+  margin: 1rem;
+}
+
+footer {
+  color: white;
+  background-color: #23272a;
```
Comment (haarsh24) [inline HomePage/style.css:176]: Use same color codes everywhere.

File: loginPage.css [hunk 9]
```diff
@@ -0,0 +1,81 @@
+body {
+    font-family: "Quicksand", sans-serif;
+    margin: 0 auto;
+    height: 100vh;
+    display: flex;
+    background: linear-gradient(to bottom right, #c01459 0%, #1627be 100%) 100% no-repeat;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```
Comment (haarsh24) [inline loginPage.css:9]: Same here. Use can add these in util.

File: loginPage.css [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+.login-container {
+    width: 60%;
+    max-width: 600px;
+    border: 1px solid white;
+    border-radius: 1rem;
+    background-color: whitesmoke;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
```
Comment (haarsh24) [inline loginPage.css:24]: Same here

**Review by haarsh24 — CHANGES_REQUESTED** [review]
Suggested some changes. After that you can merge

File: HomePage/style.css [hunk 11]
```diff
@@ -0,0 +1,196 @@
+@import url("https://dev-u-i.netlify.app/style.css");
+
+body {
+  font-family: "Quicksand", sans-serif;
+  font-size: 20px;
+  margin: 0rem;
```
Comment (nyyshaaa) [inline HomePage/style.css:6]: margin 0 will also work.

File: HomePage/index.html [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+          <img class="category-img" src="./images/shirt.jpg" />
+        </div>
+        <p>Shirts</p>
+      </div>
```
Comment (nyyshaaa) [inline HomePage/index.html:76]: you can give image styles in the css file instead of inline.

File: HomePage/index.html (hunk 1 above)
Comment (surya-nemiwal) [reply HomePage/index.html:7]: okay

File: HomePage/style.css [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+  border-radius: 0.3rem;
+  justify-content: space-between;
+  z-index: 1;
+  padding: .5rem 0rem;
+}
+
+header h1 {
+  font-size: 40px;
+  margin: 0px 20px 0px 20px;
+}
```
Comment (nyyshaaa) [inline HomePage/style.css:28]: similarly here,when you use 0 don't give it unit ,it would work without it.

File: HomePage/style.css [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+
+.card{
+  margin: 1rem;
+}
+
+footer {
+  color: white;
+  background-color: #23272a;
+  text-align: center;
```
Comment (nyyshaaa) [inline HomePage/style.css:177]: try to put these colors as variables.

**Review by nyyshaaa — APPROVED** [review]
looks good to be merged, suggested few things which you can look into later .

File: HomePage/index.html [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+        </li>
+        <li>
+          <a href="" alt="gmail">
+            <i
+              ><i><img src="./images/gmail.svg" /></i
+            ></i>
+          </a>
+        </li>
+        <li>
+          <a href="" alt="twitter">
```
Comment (Sharmaryan) [inline HomePage/index.html:125]: href is blank, instead you can set its value # as of now

File: HomePage/style.css [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+  font-size: 20px;
+  margin: 0rem;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0.5rem;
+  left: .5rem;
```
Comment (Sharmaryan) [inline HomePage/style.css:14]: for less than 1rem you can go with px (upto 10px)

File: HomePage/style.css [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+  text-align: center;
+  padding: 2rem;
+  max-width: 30rem;
+  margin: auto;
+  border-radius: 1rem;
+}
+
+.price-detail ul {
+  list-style-type: none;
+  padding: 0px;
```
Comment (Sharmaryan) [inline HomePage/style.css:157]: no need to write px, rem , or any unit after 0

File: ProductListingPage/productListing.html [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Quicksand:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../HomePage/style.css" />
+    <title>Document</title>
```
Comment (Sharmaryan) [inline ProductListingPage/productListing.html:17]: fix title

File: ProductListingPage/productListing.html [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+        <hr>
+        <ul><h3>Category</h3>
+            <li><input type="checkbox" name="" id="">Cotton</li>
+            <li><input type="checkbox" name="" id="">Nylon</li>
+            <li><input type="checkbox" name="" id="">Rayon</li>
+        </ul>
+        <hr>
+        <ul><h3>Rating</h3>
+            <li><input type="radio" name="rating" id="">5 Star</li>
+            <li><input type="radio" name="rating" id="">4 Star and above</li>
```
Comment (Sharmaryan) [inline ProductListingPage/productListing.html:71]: id's are empty

File: wishlist/wishlist.html [hunk 20]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Sharmaryan) [inline wishlist/wishlist.html:6]: again fix title

**Review by Sharmaryan — CHANGES_REQUESTED** [review]

**Review by Sharmaryan — APPROVED** [review]
UI is nice and I have suggested few changes

**Pull request merged**